versioned_docs/version-latest/protocol/wallets/polkadot.md

```diff
@@ -56,15 +61,15 @@ Some users may be provided an existing mnemonic seed phrase that may have been p
 
 2. Once extension is installed, Open it and click `+`, & `Import account from pre-existing seed`
 
-    ![step-2a](/img/doc-imgs/polkadot/step-2a.png)
+![step-2a](/img/doc-imgs/polkadot/step-2a.png)
```

Comment:
I see the Tabs(Spaces) are removed from the images, initially they were present so the image would be nested under the number list. Very minor detail, but I thought was worth mentioning 😄

Reply:
I agree that it looks better, but especially when used with such things as :::tip or :::note etc, it often crashes the structure of the page. Example on prod: https://docs.subspace.network/docs/protocol/wallets/subwallet

Reply:
So I'm kind of doubting whether nesting worth it. Wdyt?

Reply:
Ah I see, makes sense to me! In that case feel free to disregard my initial comment 😄